doc/sphinx/source/conf.py

```diff
@@ -39,7 +39,7 @@
 # ones.
 extensions = [
     'breathe',
-    #'hoverxref.extension',  # still in beta; not rendering well on RTD
+    # 'hoverxref.extension',  # still in beta; not rendering well on RTD
```

Comment:
I'm not sure why my `make style` wanted this.

Reply:
Maybe it's the comment style that requires having at least a space after the comment symbol. I wouldn't worry too much about it now, since it most likely change once we finalize the use of the hoverxf extension.